### PR TITLE
PRS in Tanasee1 - no. 6

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
PRS from Tanasee1 as discussed and revised in Pull request #742.

### CODE REVIEW

File: new/PRS14098DabraSyon.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <change who="CH" when="2023-10-06">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደብረ፡ ጽዮን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dabra Ṣəyon</persName>
+                    <occupation type="ecclesiastic">Qis Gabaz</occupation>
```
Comment (eu-genia) [inline new/PRS14098DabraSyon.xml:50]: why capitalized?

File: new/PRS14100ZaraKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘርአ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zarʾa Krəstos</persName>
+                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <occupation type="ecclesiastic">rāq masrā</occupation>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14100ZaraKrestos" passive="Tanasee1#e18 #e19"/>
```
Comment (eu-genia) [inline new/PRS14100ZaraKrestos.xml:54]: here and everywhere - you may not have multiple passive members, and you may not have a member starting with `#` as this refers to the record in question (e.g.` #e19` here means `PRS14100ZaraKrestos.xm#e19`, something that does not exist, would result in an error.
one could potentially list `Tanasee1#e18 Tanasee1#e19
` but the app cannot handle this, so the rule is two passive members = two relations.

File: new/PRS14101Baselyos.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ባስልዮስ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Bāsəlyos</persName>
+                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <occupation type="ecclesiastic">liqa diyāqonat</occupation>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14101Baselyos" passive="Tanasee1#e18 #e19"/>
```
Comment (eu-genia) [inline new/PRS14101Baselyos.xml:54]: as above

File: new/PRS14098DabraSyon.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                    <persName><roleName type="title">ʾabba</roleName></persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14098DabraSyon" passive="Tanasee1#e18 #e19"/>
```
Comment (eu-genia) [inline new/PRS14098DabraSyon.xml:54]: here and everywhere - you may not have multiple passive members, and you may not have a member starting with # as this refers to the record in question (e.g. #e19 here means PRS14100ZaraKrestos.xm#e19, something that does not exist, would result in an error.
one could potentially list Tanasee1#e18 Tanasee1#e19  but the app cannot handle this, so the rule is two passive members = two relations.

File: new/PRS14097Hirut.xml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ኂሩት</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Ḫirut</persName>
+                    <persName><roleName type="title">mamhər</roleName></persName>
+                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS14097Hirut" passive="Tanasee1#e18 #e19"/>
```
Comment (eu-genia) [inline new/PRS14097Hirut.xml:54]: here and everywhere - you may not have multiple passive members, and you may not have a member starting with # as this refers to the record in question (e.g. #e19 here means PRS14100ZaraKrestos.xm#e19, something that does not exist, would result in an error.
one could potentially list Tanasee1#e18 Tanasee1#e19  but the app cannot handle this, so the rule is two passive members = two relations.

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14100ZaraKrestos.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14100ZaraKrestos.xml:54]: I resolved this problem by deleting the reference to the specific extras and kept just alone the reference to Tanasee1.

File: new/PRS14101Baselyos.xml [hunk 6]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ባስልዮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Bāsəlyos</persName>
-                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14101Baselyos.xml:50]: here it was actually correct - if you do not have the `lang `then you do not need the `type`

File: new/PRS14099Malkasedeq.xml [hunk 7]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">መልከጼዴቅ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Malkaṣedeq</persName>
-                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14099Malkasedeq.xml:50]: you could have left it. without `xml:lang` you do not need the `type`

File: new/PRS14098DabraSyon.xml [hunk 8]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ደብረ፡ ጽዮን፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Dabra Ṣəyon</persName>
-                    <occupation type="ecclesiastic">Qis Gabaz</occupation>
-                    <persName><roleName type="title">ʾabba</roleName></persName>
+                    <occupation type="ecclesiastic">qes gabaz</occupation>
```
Comment (eu-genia) [inline new/PRS14098DabraSyon.xml:50]: you could have left it. without `xml:lang` you do not need the `type`

File: new/PRS14093Zayohannes.xml [hunk 9]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ዘዮሐንስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Zayoḥannəs</persName>
-                    <persName><roleName type="title">ʾabuna</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabuna</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14093Zayohannes.xml:50]: you could have left it. without `xml:lang` you do not need the `type`

File: new/PRS14097Hirut.xml [hunk 10]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ኂሩት</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Ḫirut</persName>
-                    <persName><roleName type="title">mamhər</roleName></persName>
-                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">mamhər</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14097Hirut.xml:50]: you could have left it. without `xml:lang` you do not need the `type`

File: new/PRS14100ZaraKrestos.xml [hunk 11]
```diff
@@ -47,11 +47,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ዘርአ፡ ክርስቶስ፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Zarʾa Krəstos</persName>
-                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14100ZaraKrestos.xml:50]: you could have left it. without xml:lang you do not need the type

**Review by eu-genia — APPROVED** [review]

File: new/PRS14098DabraSyon.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14098DabraSyon.xml]: I changed it to "qes gabaz".

File: new/PRS14101Baselyos.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14101Baselyos.xml]: I eliminated the reference to the specific extras/additions.

File: new/PRS14098DabraSyon.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14098DabraSyon.xml]: Yes, done.

File: new/PRS14097Hirut.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14097Hirut.xml]: Yes. Resolved.

File: new/PRS14101Baselyos.xml (hunk 6 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14101Baselyos.xml:50]: OK. I will remember in the future.

**Comment by CarstenHoffmannMarburg** [comment]
If all requests are solved, can this branch be merged?

**Comment by eu-genia** [comment]
sure, if all have approved it can be merged, you can also do it when it is green

**Pull request merged**